services/movie_session.py

```diff
@@ -0,0 +1,46 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+
+    return MovieSession.objects.create(show_time=movie_show_time,
+                                       movie_id=movie_id,
+                                       cinema_hall_id=cinema_hall_id)
```

Comment:
Ensure that `movie_show_time` is a `datetime` object, not a string, to match the `DateTimeField` in the model.